src/scales/scale.time.js

```diff
@@ -336,8 +336,33 @@ module.exports = function(Chart) {
 		},
 		ticks: {
 			autoSkip: false,
-			mode: 'linear',   // 'linear|series'
-			source: 'auto'    // 'auto|labels'
+
+			/**
+			 * Ticks distribution along the scale:
+			 * - 'linear': ticks are spread according to their time (distances can vary),
```

Comment:
Or rather maybe more informative to say "ticks and data"

When `source:auto` mode, we try to place ticks spread evenly regardless of `mode` so I think it helps explain it better if we mention data as well